src/main/java/de/zalando/aruha/nakadi/domain/EventType.java

```diff
@@ -2,30 +2,60 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.google.common.collect.Lists;
 
 public class EventType {
```

Comment:
That's much better now :)
I think that actually  @JsonProperty("owning_application") could be also removed, as we defined the default naming strategy for jackson as a snake-case (see here: https://github.com/zalando/nakadi/blob/nakadi-jvm/src/main/java/de/zalando/aruha/nakadi/config/NakadiConfig.java#L75), so it should produce snake case by default already.